wagtail/test/settings.py

```diff
@@ -70,10 +71,10 @@
         "BACKEND"
     ] = "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
 
-    # DJANGO_VERSION < 4.2
-    STATICFILES_STORAGE = (
-        "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
-    )
+    if DJANGO_VERSION < (4, 2):
+        STATICFILES_STORAGE = (
+            "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
```

Comment:
Probably out of scope for this PR, but should we be updating this [in the project template too](https://github.com/wagtail/wagtail/blob/6ef8da04eb9141d8bc43bcbbce65d9d80f4dac2d/wagtail/project_template/project_name/settings/base.py#L140-L143)?

Reply:
Ah, yep, thanks! I'll update it in a separate PR.